src/Orchard.Cms.Web/project.json

```diff
@@ -37,7 +43,7 @@
     }
   },
   "frameworks": {
-    "netcoreapp1.1": {
+    "netcoreapp1.0": {
```

Comment:
What happens when you bump it up to `netcoreapp1.1`? It should work just fine. See https://github.com/aspnet/live.asp.net/blob/dev/src/live.asp.net/project.json#L68